source/family/maxim/max32690/target.c

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file    target.c
+ * @brief   Target information for the MAX32690
+ *
+ * DAPLink Interface Firmware
+ * Copyright (c) 2023 Analog Devices, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "target_config.h"
+
+#include "flash_blob.c"
+
+/* MAX32690 3MiB+256KB Flash, 1MiB RAM */
+target_cfg_t target_device = {
+    .sectors_info                   = sectors_info,
+    .sector_info_length             = (sizeof(sectors_info))/(sizeof(sector_info_t)),
+    .flash_regions[0].start         = 0x10000000,
+    .flash_regions[0].end           = 0x10000000 + MB(3) + KB(256),
+    .flash_regions[0].flags         = kRegionIsDefault,
+    .flash_regions[0].flash_algo    = (program_target_t *) &flash,
+    .ram_regions[0].start           = 0x20000000,
+    .ram_regions[0].end             = 0x20000000 + MB(1),
+    .target_vendor                  = "ADI",
```

Comment:
This should be `"Maxim"` to match the CMSIS Pack (https://www.keil.arm.com/packs/max32690-maxim/)

Reply:
Thanks, updated.